client/src/utils/ajax.ts

```diff
@@ -209,6 +213,10 @@ export function postResetProgress(): Promise<void> {
   return post('/account/reset-progress', {});
 }
 
+export function postWebhookToken(): Promise<void> {
+  return post('/user/webhook-token', {});
```

Comment:
```suggestion
  return post('/user/webhook-token');
```
Is the empty body necessary here?

Reply:
Currently, they sort of are - the helper functions at the top of this file expect two arguments. We could make the second one (body) optional. There were several other uses of these functions that used an empty object as the second argument, so I did what those were doing. Any preference?

Reply:
I think, since there's no need to add a body just because you're POSTing, that we should make it optional.  I'll create a note for myself to change this after we merge, though, what you've done seems good for this PR.